src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -518,33 +518,37 @@ public static int findRepeatedSubstring(byte[] bases) {
      * @return                       Number of repetitions (0 if testString is not a concatenation of n repeatUnit's
```

Comment:
Add a better explanation for `lookForward` mode vs. backward mode (I was initially confused as to the intended behavior of this method in backward mode), plus an example for the backward case.